include/sys/util.h

```diff
@@ -131,6 +131,11 @@ extern "C" {
 #define CONTAINER_OF(ptr, type, field) \
 	((type *)(((char *)(ptr)) - offsetof(type, field)))
 
+/**
+ * @brief Returns rounded division of @p A divided by @p B.
+ */
+#define ROUNDED_DIV(A, B) (((A) + ((B) / 2)) / (B))
```

Comment:
This evaluates the argument B twice. Therefore I think this should be a static inline function, or have a similar note to `CLAMP`.